package/test/simple_beam_location_test.dart

```diff
@@ -55,7 +59,7 @@ void main() {
       delegate.beamToNamed('/unknown');
       expect(delegate.currentBeamLocation, isA<NotFound>());
 
-      delegate.beamToNamed('/test/unknown');
+      delegate.beamToNamed('/Test/unknown');
```

Comment:
Why is it a capital T now?

Reply:
This test was failing because I had a route with this regex `/path-param/(?<test>[a-z]+` and it has a match when applied on `/test/unknown` so I had to change it.